swarm/pss/prox_test.go

```diff
@@ -255,93 +243,153 @@ func testProxNetwork(t *testing.T) {
 		// context deadline exceeded
 		// however, it might just mean that not all possible messages are received
 		// now we must check if all required messages are received
-		cnt := tstdata.getMsgCount()
-		log.Debug("TestProxNetwork finished", "rcv", cnt)
-		if cnt < tstdata.requiredMessages {
+		log.Debug("TestProxNetwork finished", "rcv", tstdata.getMsgCount())
+		if !isSuccess(tstdata) {
 			t.Fatal(result.Error)
 		}
 	}
-	t.Logf("completed %d", result.Duration)
 }
 
-func (tstdata *testData) sendAllMsgs() {
+func isSuccess(d *testData) bool {
+	cnt := d.getMsgCount()
+	if cnt < d.requiredMessages {
+		if d.isExpectationsChanged() {
+			// network configuration has changed since beginning of
```

Comment:
why would this happen  in this  test? it really should not

Reply:
@frncmx yes

Reply:
@zelig should i remove it?

Reply:
sure please